### PR TITLE
Prevent adding session param to query string when redirecting to exte…

### PR DESCRIPTION
**Pull request by fmnoise**
…rnal url

### CODE REVIEW

File: lib/cookieless.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def call(env)
         if env['action_dispatch.request.path_parameters'] && %w(css js xml).exclude?(env['action_dispatch.request.path_parameters'][:format].to_s)
           session_id = save_cookies_by_session_id(session_id || rack_session_id(env), env, header["Set-Cookie"])
           ## fix 3xx redirect
-          header["Location"] = convert_url(header["Location"], session_id) if header["Location"]
+          header["Location"] = convert_url(header["Location"], session_id) if header["Location"] && !external_url?
```
Comment (adie) [inline lib/cookieless.rb]: Why don't you pass any parameters here?

**Comment by adie** [comment]
Would be actually good to have some tests for it, not sure if it will take to much time to setup testing.

**Comment by cheef** [comment]
I think we need a test for that, also it's good to send fork to original repository with this fix

File: Appraisals [hunk 2]
```diff
@@ -0,0 +1,11 @@
+appraise '3.1' do
+  gem 'rails', '~> 3.1.0'
+end
+
+appraise '3.2' do
+  gem 'rails', '~> 3.2.0'
+end
+
+appraise '4.0' do
+  gem 'rails', '~> 4.0.0'
```
Comment (adie) [inline Appraisals]: maybe better test it with Rails 4.2?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+rvm:
+  - 1.9.3
+  - 2.0.0
+  - 2.1.0
+  - 2.1.1
+  - rbx
+gemfiles:
+  - gemfiles/3.1.gemfile
+  - gemfiles/3.2.gemfile
+  - gemfiles/4.0.gemfile
```
Comment (adie) [inline .travis.yml]: Do you still need this file as we run tests in CircleCI?

**Comment by galaydaroman** [comment]
Good for me

**Pull request merged**